### DISCUSSION TITLE
how to install antd design on next js 13 version for new app directory ?

### DISCUSSION THREAD

**Discussion by 19narekHarutyunyan97** [ant-design/ant-design#41097] [General]
Can I install ant design 5 on next js 13 version ? if I can please ask how ? ))

**Comment by bombillazo** [comment]
It's tricky. Here is the example repo:
https://github.com/ant-design/create-next-app-antd/tree/main

And the SSR compatibility guide:
https://ant.design/docs/react/customize-theme#server-side-render-ssr

Basically you need to either prebake styles or use an inline approach if you want SSR to work with Ant Design

**Reply by akashi7** [reply]
@bombillazo  instead of using inline how about using tailwind ?

**Comment by marufprocode** [comment]
I am facing the same problem, can not import antd components in the server components! have you got any solutions?
